### PR TITLE
Handled cluster context switching issue in `IssueMultipleRestoresWithNamespaceAndStorageClassMapping`

### PR DESCRIPTION
**Pull request by mkoppal-px**
**What this PR does / why we need it**:

Handled cluster context switching issue in `IssueMultipleRestoresWithNamespaceAndStorageClassMapping`.
We observed in this [jenkins run](https://jenkins.pwx.dev.purestorage.com/job/portworx-backup/job/system-tests/job/px-backup-system-test-vanilla/170/consoleFull) that if something goes wrong while we have switched context to destination cluster, the `JustAfterEach` is not handling it properly in `IssueMultipleRestoresWithNamespaceAndStorageClassMapping`.

Added the first line in `JustAfterEach` to switch to source cluster and proceed with cleanup.

**Which issue(s) this PR fixes** (optional)
Closes #PA-880

**Special notes for your reviewer**:

### CODE REVIEW

File: tests/backup/backup_share_test.go [hunk 1]
```diff
@@ -3617,46 +3617,48 @@ var _ = Describe("{IssueMultipleRestoresWithNamespaceAndStorageClassMapping}", f
...
 		opts := make(map[string]bool)
 		opts[SkipClusterScopedObjects] = true
 		ValidateAndDestroy(contexts, opts)
 		log.InfoD("Deleting the backup created")
 		backupDriver := Inst().Backup
+		ctx, err := backup.GetAdminCtxFromSecret()
+		log.FailOnError(err, "Fetching px-central-admin ctx")
 		backupUID, err := backupDriver.GetBackupUID(ctx, backupName, orgID)
 		dash.VerifySafely(err, nil, fmt.Sprintf("Getting the backup UID for %s", backupName))
 		_, err = DeleteBackup(backupName, backupUID, orgID, ctx)
```
Comment (kphalgun-px) [inline tests/backup/backup_share_test.go:3633]: I think CleanupCloudSettingsAndClusters will take care of deleting backups.

**Review by kphalgun-px — APPROVED** [review]
LGTM

**Review by snigdha-px — APPROVED** [review]
LGTM

**Review by kshithijiyer-px — APPROVED** [review]
LGTM

**Review by vpinisetti-px — APPROVED** [review]
LGTM

**Review by lsrinivas-pure — APPROVED** [review]
lgtm

**Pull request merged**